packages/react/package.json

```diff
@@ -40,8 +40,8 @@
     "prerelease": "gitpkg publish",
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
-    "cypress:open": "cypress open",
-    "cypress:run": "cypress run"
+    "cypress:open": "TZ=UTC cypress open",
+    "cypress:run": "TZ=UTC cypress run"
```

Comment:
Our CI uses UTC timezone, so setting the timezone here can prevent us from having a different result in development and CI.